Update documentation cross references (#327)
OpenMS/pyopenms-docs: docs/source/id_by_mz.rst

Feature Detection
*****************
in: path to centroid mzML files (files)

> out: list with :py:class:`~.FeatureMaps` (feature_maps)

Reviewer: ```suggestion
out: list of :py:class:`~.FeatureMap` (feature_maps)
```

Feature Map Retention Time Alignment
************************************

> in: unaligned :py:class:`~.FeatureMaps` (feature_maps)

Reviewer: ```suggestion
in: unaligned list of :py:class:`~.FeatureMap` (feature_maps)
```

> out: :py:class:`~.FeatureMaps` aligned on the first feature map in the list (feature_maps)

Reviewer: ```suggestion
out: list of :py:class:`~.FeatureMap` aligned to the first feature map in the list (feature_maps)
```

Feature Linking
***************

> in: list with :py:class:`~.FeatureMaps` (feature_maps)

Reviewer: ```suggestion
in: list of:py:class:`~.FeatureMap` (feature_maps)
```
